src/main/java/com/azure/cosmos/sample/async/AsyncMain.java

```diff
@@ -52,14 +55,14 @@ public static void main(String[] args) {
         AsyncMain p = new AsyncMain();
 
         try {
-            System.out.println("Starting ASYNC main");
+            logger.info("Starting ASYNC main");
             p.getStartedDemo();
-            System.out.println("Demo complete, please hold while resources are released");
+            logger.info("Demo complete, please hold while resources are released");
         } catch (Exception e) {
             e.printStackTrace();
-            System.err.println(String.format("Cosmos getStarted failed with %s", e));
+            logger.error("Cosmos getStarted failed with %s", e);
```

Comment:
the place holder symbol for logger is "{}" so in case you have an arg you should do `logger.error("xyz {}", count)` 

however in this case you are passing an exception and as long as exception is the last arg, slf4j understands it without any need for "{}" place holder so this need to change to following.

```suggestion
            logger.error("Cosmos getStarted failed with", e);
```